### PR TITLE
feat: add new api for query and delete clientid

### PR DESCRIPTION
**Pull request by azam-ismail**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -55,6 +55,7 @@ LOGGING_INFO_ENABLE=false
 INTERNAL_TOPIC=internal
 NATS_MAX_AGE=86400000
 NATS_MAX_SIZE=8388608
+NATS_MAX_CLIENT_ID=50
```
Comment (kim-energyweb) [inline src/main/resources/application.properties:58]: Return the value in `GET /config` so Client GW can reference this.

**Review by kim-energyweb — CHANGES_REQUESTED** [review]

File: src/main/java/org/energyweb/ddhub/dto/ClientDTO.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class ClientDTO {
+
+	@Valid
+    @NotNull
+    @NotEmpty
+    private Set<@NotNull @NotEmpty String> clientIds;
```
[the remark this answers is not in the document]
Comment (azam-ismail) [reply src/main/java/org/energyweb/ddhub/dto/ClientDTO.java:21]: yup , but this are for delete clientid.

File: src/main/resources/application.properties (hunk 1 above)
Comment (azam-ismail) [reply src/main/resources/application.properties:58]: ok

File: src/main/java/org/energyweb/ddhub/dto/ClientDTO.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (kim-energyweb) [reply src/main/java/org/energyweb/ddhub/dto/ClientDTO.java:21]: ok, understood.

File: src/main/resources/application.properties (hunk 1 above)
Comment (azam-ismail) [reply src/main/resources/application.properties:58]: updated

**Review by kim-energyweb — APPROVED** [review]
LGTM

**Pull request merged**